src/features/defi/providers/foxy/components/FoxyManager/Withdraw/components/Withdraw.tsx

```diff
@@ -168,41 +144,17 @@ export const Withdraw: React.FC<
         payload: true,
       })
       try {
-        // Check is approval is required for user address
-        const _allowance = await foxyApi.allowance({
-          tokenContractAddress: rewardId,
-          contractAddress,
-          userAddress: accountAddress,
+        const estimatedGasCryptoBaseUnit = await getWithdrawGasEstimateCryptoBaseUnit(formValues)
+        if (!estimatedGasCryptoBaseUnit) return
+        dispatch({
+          type: FoxyWithdrawActionType.SET_WITHDRAW,
+          payload: { estimatedGasCryptoBaseUnit },
+        })
+        onNext(DefiStep.Confirm)
+        dispatch({
+          type: FoxyWithdrawActionType.SET_LOADING,
+          payload: false,
         })
-
-        const allowance = bnOrZero(bn(_allowance).div(bn(10).pow(asset.precision)))
-
-        // Skip approval step if user allowance is greater than or equal requested deposit amount
```

Comment:
This was boilerplate from deposit that ended up being used at withdraw step, making useless XHRs in the process.
FOXy withdraw doesn't need an approval as opposed to withdrawing LPs where you need to approve the LP token - grep for `safeTransfer` in https://github.com/shapeshift/yieldies/blob/main/src/contracts/Staking.sol as a sanity check